.husky/task-runner.json

```diff
@@ -0,0 +1,40 @@
+{
+   "$schema": "https://alirezanet.github.io/Husky.Net/schema.json",
+   "tasks": [
+      {
+         "name": "commit-message-linter",
+         "command": "dotnet",
+         "args": [
+            "husky",
+            "exec",
+            ".husky/csx/commit-lint.csx",
+            "--args",
+            "${args}"
+         ]
+      },
+      {
+         "name": "csharpier",
+         "group": "pre-commit",
+         "command": "dotnet",
+         "args": ["csharpier", "${staged}"],
+         "include": ["**/*.cs"]
+      },
+      {
+         "name": "warning-check",
+         "command": "dotnet",
+         "group": "pre-push",
+         "args": ["build", "/warnaserror"],
+         "include": ["**/*.cs", "**/*.vb"]
+      },
+      {
+         "name": "run-jb-clean-up-code",
+         "command": "sh",
```

Comment:
Can this one also be invoked via `dotnet` command like the others? It gives consistency and also ensure that things remain Windows-friendly